kasa/experimental/smartcamera.py

```diff
@@ -0,0 +1,79 @@
+"""Module for smartcamera."""
```

Comment:
Just a general note, should we ignore experimental from codecov, or would you mind adding some tests here?

Reply:
Yes agreed, let's exclude `experimental` from codecov for now and add tests when we move it out.

Reply:
As a counter argument, having test coverage also for these pieces would be a good idea. We may want to ship a release before everything is moved out from it, and we still want to have tests for the library code we are shipping. What do you think?

Reply:
I’d rather get it merged so more “beta”  users can test and then we can add `dump_devinfo` and get fixtures before we start adding proper tests.

Reply:
Ok, and I think we do not currently have much pressure to do releases anyway, so I agree that it's fine like that. However, on second thought, I think we should keep it simple and avoid doing any explicit exclusions for this, but just rather ignore the codecov warnings on the affected PRs.